Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM python:${PYTHON_VERSION}
 WORKDIR /src
 COPY . .
 
-ARG VERSION
+ARG VERSION=0.0.0-dev
```

Comment:
I chose `0.0.0-dev`, but not sure if anything uses the version, so if `99.9.9-dev` or `99.0.0-dev` ("always higher than what exists") would be better.

Happy to change!

Reply:
Okay either way, but maybe someone with more python experience than me can chime in.

Reply:
@neersighted any preference here?

Reply:
[PEP-440](https://peps.python.org/pep-0440/) defines the syntax for declaring a development package as the `.devN` suffix (where N is just a number), so I'd use that instead of `-dev`.

`pip` __should__ only install stable packages when dependencies are declared as `someDep >= 1.2.3`, but it will allow installation of `.devN` dependencies if by chance the user has a dependency declared as `someDep >= 4.5.6.dev2` for example.

Because of ☝️, I'd suggest staying with the lower v `0.0.0.dev` just to be on the safer side in case we accidentally end up publishing a package with that version to pypi.  

I'm not 100% sure this is the best approach though, so if others have more experience please chime in 😄

Reply:
If it expects a number after `.dev`, I can make it `0.0.0.dev0` I guess @neersighted SGTY?

Reply:
`0.0.0.dev0` is PEP 440 (though the current spec lives at https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers) compliant, so that works for me.

Notably all numeric components are allowed to be 0; the proposed version does pass Poetry's PEP 440 validation, which is notoriously a bit strict.